osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -173,10 +180,15 @@ protected override void ApplySkin(ISkinSource skin, bool allowFallback)
 
             Body.AccentColour = skin.GetConfig<OsuSkinColour, Color4>(OsuSkinColour.SliderTrackOverride)?.Value ?? AccentColour.Value;
             Body.BorderColour = skin.GetConfig<OsuSkinColour, Color4>(OsuSkinColour.SliderBorder)?.Value ?? Color4.White;
+
+            allowSliderBallTint = skin.GetConfig<OsuSkinConfiguration, bool>(OsuSkinConfiguration.AllowSliderBallTint)?.Value ?? false;
+            updateSliderBallTint();
         }
 
         private void updatePathRadius() => Body.PathRadius = slider.Scale * sliderPathRadius;
 
+        private void updateSliderBallTint() => Ball.Colour = (alwaysTintSliderBall.Value | allowSliderBallTint) ? AccentColour.Value : Color4.White;
```

Comment:
Was bitwise or (`|`) instead of a logical or (`||`) intended here? This might actually be a bit nicer with `||` due to the short-circuiting laws.